### DISCUSSION TITLE
3.6.0

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#53] [Announcements]
Blender 3.6.0, commit [c7fc78b81ecb](https://projects.blender.org/blender/blender/commit/c7fc78b81ecb).
Build environment:

    Visual Studio 2019, version 16.11.27
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/3.6.0'>3.6.0</a>.</em>

**Comment by LIONGAMESS** [comment]
Holaa em, me lanza un error y nose que hacer, y cuando le doy en continue anyway se cierra, que hago?
![image](https://github.com/nalexandru/BlenderCompat/assets/138704972/4c472c2b-a615-40e2-82af-8967e00b79b1)

**Reply by nalexandru** [reply]
This could be related to #15. You can try updating the graphics driver.
The last Intel driver for Windows 7 can be found here:
https://www.intel.com/content/www/us/en/download/18424/intel-graphics-driver-for-windows-7-8-1-15-36.html
I'm not sure if that is the correct driver for your GPU; if it doesn't want to install then try this one:
https://www.intel.com/content/www/us/en/download/18369/intel-graphics-driver-for-windows-15-40.html

**Reply by LIONGAMESS** [reply]
no funciono T vT no son compatibles con mi pc los drivers esos

**Comment by LIONGAMESS** [comment]
muchas gracias, intentare ver si funciona
